### PR TITLE
Pre final branch

### PR DESCRIPTION
**Pull request by 1khansaad**

### CODE REVIEW

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]
LGTM

File: style.css [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+  font-size: 30px;
+}
+.size,
+.color {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  margin-bottom: 5px;
```
Comment (syedabdullahali) [inline style.css:123]: use shorthand property

**Review by syedabdullahali — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (nehalskdev) [inline style.css:123]:
```suggestion
  margin: 0 0 5px 0;
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  display: inline-block;
+
+  text-decoration: none;
+
+  background: #999;
+  border: 1px solid blue;
+  color: black;
+
+  padding: 4px;
+  margin-top: 7px;
```
Comment (nehalskdev) [inline style.css:174]:
```suggestion
  margin: 7px 0 0 0;
```

**Review by nehalskdev — CHANGES_REQUESTED** [review]
Make changes as suggested

**Pull request merged**